climpred/smoothing.py

```diff
@@ -0,0 +1,157 @@
+import xarray as xr
+
+try:
+    import xesmf as xe
+
+    def spatial_smoothing_xesmf(ds, boxsize=(5, 5)):
+        """Apply spatial smoothing by regridding to `boxsize` grid.
+
+        Reference:
+          * Goddard, L., A. Kumar, A. Solomon, D. Smith, G. Boer, P.
+                Gonzalez, V. Kharin, et al. “A Verification Framework for
+                Interannual-to-Decadal Predictions Experiments.” Climate
+                Dynamics 40, no. 1–2 (January 1, 2013): 245–72.
+                https://doi.org/10/f4jjvf.
+
+        Args:
+            ds (xr.object): input. xr.DataArray prefered.
+            boxsize (tuple): target grid.
+                             Defaults to `(5, 5)` (see Goddard et al. 2013).
+
+        Returns:
+            ds_smoothed (xr.object): boxsize-regridded input
+
+        """
+        if isinstance(ds, xr.Dataset):
+            was_dataset = True
+            ds = ds.to_array().squeeze()
+        else:
+            was_dataset = False
+        ds_out = xe.util.grid_global(*boxsize)
+        regridder = xe.Regridder(ds, ds_out, 'bilinear')
+        res = regridder(ds)
+
+        if was_dataset:
+
+            def _variable_dataarray_to_dataset(da):
```

Comment:
wrapper to repack a dataset which was decomposed into variables via `ds.to_array()`

Reply:
Not necessary anymore I think